build.gradle

```diff
@@ -57,6 +57,7 @@ allprojects {
 		minecraft "com.mojang:minecraft:$Globals.mcVersion"
 		mappings "net.fabricmc:yarn:${Globals.mcVersion}${Globals.yarnVersion}:v2"
 		modCompile "net.fabricmc:fabric-loader:0.7.2+build.174"
+		compileOnly "org.checkerframework:checker-qual:3.0.1"
```

Comment:
I dont think we really want this here. 

__IF__ these annotations are really wanted they should be added via loom so everyone gets them. Minecraft uses the jsr ones, so ideally we would use those imo. Or if the checker ones are really wanted we need to remap the ones in minecraft to the checker ones.

I really do not like the idea of adding them this way as most people will not have these annotations in their dev env.

Reply:
The main issue with the JSR ones is that they're broken on Java 9 and above.

Reply:
Yes. I do think the best solution would be to use checker for everything, and remap them in MC.

Reply:
Hmm, ide should be able to identify these values correctly despite that they may be absent in the dependencies. Or is there any configuration that exposes this compile-only dependency, I wonder?

For remapping vanilla annotations, I object. Vanilla has quite a few places of improper annotations, so I'd prefer annotating with a tool like enigma. FabricMC/yarn#1006

Reply:
Unfortunately, I don't think we have a way to properly expose checker framework with `compileOnly`. Maybe I should just use `compile`/`api` instead? https://docs.gradle.org/current/userguide/img/java-library-ignore-deprecated-main.png